src/main/protobuf/data_point_insertion_request.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
```

Comment:
It would be better to include them directly from https://github.com/cognitedata/protobuf-files with a git submodule instead of copying them, but we can fix that later. I've created https://cognitedata.atlassian.net/browse/DT-954 (which is not urgent and we shouldn't worry about it for now).